site2/docs/reference-pulsar-admin.md

```diff
@@ -2423,167 +2423,167 @@ $ pulsar-admin topics remove-delayed-delivery tenant/namespace/topic
 ```
 
 ### `get-max-producers`
-Get max-Producers for a topic.
+Get the maximum number of Producers for a topic.
 
 Usage
 ```bash
 $ pulsar-admin topics get-max-producers tenant/namespace/topic
 ```
 
 ### `set-max-producers`
-Set max-Producers for a topic.
+Set the maximum number of Producers for a topic.
 
 Usage
 ```bash
-$ pulsar-admin topics set-max-producers tenant/namespace/topic
+$ pulsar-admin topics set-max-producers tenant/namespace/topic options
 ```
 
 ### `remove-max-producers`
-Remove max-Producers for a topic.
+Remove the maximum number of Producers for a topic.
 
 Usage
 ```bash
 $ pulsar-admin topics remove-max-producers tenant/namespace/topic
 ```
 
 ### `get-max-consumers`
-Get max-Consumers for a topic.
+Get the maximum number of Consumers for a topic.
 
 Usage
 ```bash
 $ pulsar-admin topics get-max-consumers tenant/namespace/topic
 ```
 
 ### `set-max-consumers`
-Set max-Consumers for a topic.
+Set the maximum number of Consumers for a topic.
 
 Usage
 ```bash
-$ pulsar-admin topics set-max-consumers tenant/namespace/topic
+$ pulsar-admin topics set-max-consumers tenant/namespace/topic options
 ```
 
 ### `remove-max-consumers`
-Remove max-Consumers for a topic.
+Remove the maximum number of Consumers for a topic.
 
 Usage
 ```bash
 $ pulsar-admin topics remove-max-consumers tenant/namespace/topic
 ```
 
-### `get-max-unacked-messages-per-subscription`
-Get maxUnackedMessagesPerSubscription for a topic.
+### `get-max-unacked-messages-on-subscription`
+Get the maximum number of unacked messages per subscription for a topic.
 
 Usage
 ```bash
 $ pulsar-admin topics get-max-unacked-messages-per-subscription tenant/namespace/topic
 ```
 
-### `set-max-unacked-messages-per-subscription`
-Set maxUnackedMessagesPerSubscription for a topic.
+### `set-max-unacked-messages-on-subscription`
```

Comment:
`set-max-unacked-messages-per-subscription`

Reply:
@freeznet Can you double confirm the changes?